content/docs/command-reference/index.md

```diff
@@ -1,6 +1,10 @@
 # Using DVC Commands
 
-DVC is a command line tool. The typical DVC workflow goes as follows:
+DVC is a command line tool. For a listing of commands run `dvc -h`. All commands
+are run in the form `dvc command [arguments]`. To run any DVC command in a
+different directory, use `dvc -C <path> command`.
+
+The typical DVC workflow goes as follows:
```

Comment:
I like the new sentence "For a listing of commands run `dvc -h`." let's keep it! But that's not related to the intended change here.

Reply:
As for the -C note, it should probably not be here on the top of the doc. Let's put it after all the bullets for now? Seems like it's less important than them.

Reply:
But to really decide, I'd need to know: is it like `-V` that only exists after `dvc`? Or like `-hqr` that are in every subcommand? Thanks

Reply:
Yes the `-C <path>` flag is a modifier to `dvc` and *must* be placed before `[command]`. It actually modifies the base command class and works independently of whatever command is run, before launching the command.

I just added a little more language about comand line usage of dvc because it seemed really akward trying to just mention the new `-C` flag randomly.

So should I keep the "For a listing of commands ..." at the top, and move the whole rest of the block after the bullet points? Or put the `dvc -C <path>` sentence?

> I like the new sentence "For a listing of commands run `dvc -h`." let's keep it! But that's not related to the intended change here.

Do you want me to move this to a separate request? My main reasoning is that the section doesn't have any context to place command flags in otherwise.

Reply:
> it seemed really akward trying to just mention the new -C flag randomly.
> should I keep the "For a listing of commands ..." at the top, and move the whole rest of the block after the bullet points?

Yes, like I mentioned, let's please just keep the new "For a listing of commands run dvc -h." sentence on top, and roll back the other changes. The note on -C can be added after the bullets instead of expanding on `dvc` usage here (no need). It should probably be in an MD block quote (`>`) too, feel free to use the bulb emoji 💡 if you wish, as it's kind of a tip.

Thanks!